### PR TITLE
[Feature] - How to style a play

### LINKED ISSUE #55: ✨ [Feature request]: Add Doc on "How to add styling to your play"
### Is your feature request related to a problem? Please describe.

Currently, there is no guide on how to style the play, so the creators just create the style files and add some generic class names and override the existing styling as well.

### Describe the solution you'd like.

We should have a guide on "How to add styling to your play" where we should mention what to do and what not to do.
For Example:
- Class Names should start with the play name and in the kebab case
- Do not directly style the elements 
- And more

### Describe alternatives you've considered.

NA

### Screenshot / Screenshare

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by supminn**
- Fixes #55

### CODE REVIEW

**Review by atapas — COMMENTED** [review]
@supminn Looks good.

Can you also add a note somewhere not to override the style using !important.

**Review by atapas — APPROVED** [review]
Looks good to me.

File: docs/How-To-Guides/how-to-style-play.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# How to add styling to your Play
+
+Styling is an essential part of any play, and it's crucial to follow best practices to ensure that your code is maintainable and scalable. Here are some **dos and don'ts** to keep in mind when adding styling to your play:
+
+## Do's
+
+- Use class names that start with the play name and are in the kebab case. This naming convention helps to keep your code organized and easy to read. For example, if your play name is **"my-play"**, your class name should be `my-play__my-class`.
+- Use CSS libraries that provide additional features such as scoped styles and automatic vendor prefixing. ReactPlay has been configured to support Tailwind CSS.
```
Comment (joshi-kaushal) [inline docs/How-To-Guides/how-to-style-play.md:8]: We should add a separate point regarding the use of Tailwind CSS.

**Review by joshi-kaushal — CHANGES_REQUESTED** [review]

File: docs/How-To-Guides/how-to-style-play.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  return <div className={styles["my-play__my-class"]}>Hello World</div>;
+}
+```
+
+- Use `styled-components` to create reusable and composable components. `styled-components` allow you to write CSS code that is scoped to a specific component and can be reused across your project.
+
+## Don'ts
+
+- Do not use inline styles unless necessary. Inline styles can make your code harder to read and maintain, and they can also negatively impact performance. Instead, use CSS classes to style your components. This approach helps to keep your code organized and makes it easier to maintain.
+  For example, instead of using inline styles like this:
```
Comment (joshi-kaushal) [inline docs/How-To-Guides/how-to-style-play.md:38]: remove extra space

File: docs/How-To-Guides/how-to-style-play.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshi-kaushal) [reply docs/How-To-Guides/how-to-style-play.md]: 1. [Scope style](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope)
2. [automatic vendor prefixing](https://css-tricks.com/autoprefixer/)

Check if these links are correct and add them if they are. Let's resolve it.

**Review by joshi-kaushal — APPROVED** [review]

**Review by atapas — APPROVED** [review]

**Pull request merged**